### PR TITLE
Update class.avatar.php

### PR DESCRIPTION
**Pull request by HPringles**

### CODE REVIEW

**Comment by TheStrato** [comment]
I was wondering about that myself.

**Comment by HPringles** [comment]
So, from what I read on the gravatar docs, the file needs to be hosted and accessed through a URL, and not through a local file path, as it is sent off the gravatar who then return the file.

The simple solution is to host the file somewhere that it won't get taken down.

**Comment by HPringles** [comment]
Updated to use the new link - fully url encoded also

File: include/class.avatar.php [hunk 1]
```diff
@@ -212,7 +212,7 @@ function __construct($user, $imageset) {
         $this->email = $user->getEmail();
         // $this->d = $imageset;
         // Changed for new default avatar
-        $this->d = 'https%3A%2F%2Fstage-9.co.uk%2Fwp-content%2Fuploads%2F2017%2F12%2Fstage9projectpatch.png'
+        $this->d = 'https%3A%2F%2Fsupport.stage-9.co.uk%2Fimages%2Fstage9projectpatch.png'
```
Comment (dapperrobotbear) [inline include/class.avatar.php]: You've accidentally included the "+" at the beginning of this line.

**Comment by dapperrobotbear** [comment]
Closing in favor of the other PR from @TheStrato

**Pull request closed without merging**